boa3_test/tests/compiler_tests/test_bytes.py

```diff
@@ -1,18 +1,17 @@
-from boa3_test.tests.boa_test import BoaTest  # needs to be the first import to avoid circular imports
+from boa3_test.tests import boatestcase
 
 from boa3.internal.exception import CompilerError, CompilerWarning
 from boa3.internal.neo.vm.opcode.Opcode import Opcode
 from boa3.internal.neo.vm.type.Integer import Integer
 from boa3.internal.neo.vm.type.StackItem import StackItemType
 from boa3.internal.neo3.vm import VMState
+from boa3_test.tests.boatestcase import FaultException
 from boa3_test.tests.test_drive.testrunner.boa_test_runner import BoaTestRunner
 
 
-class TestBytes(BoaTest):
+class TestBytes(boatestcase.BoaTestCase):
     default_folder: str = 'test_sc/bytes_test'
 
-    SUBSEQUENCE_NOT_FOUND_MSG = 'subsequence of bytes not found'
```

Comment:
There is a small chance that Neo could change this error message. I think it would be better to keep this string here and use the const on the tests below.